src/draw/lv_draw_label.c

```diff
@@ -449,6 +449,9 @@ LV_ATTRIBUTE_FAST_MEM void lv_draw_letter(const lv_point_t * pos_p, const lv_are
         return;
     }
 
+    if (g.resolved_font) {
```

Comment:
It should be removed too, shouldn't it?

Reply:
This is still necessary, as when we get the bitmap, we don't get hold of glyph description. So it must use correct font pointer to get glyph bitmap, unless we change `lv_font_get_glyph_bitmap` signature or create another function with glyph info.

Reply:
True, my bad.